build.gradle

```diff
@@ -538,6 +538,8 @@ project(':core') {
     from(project(':streams').configurations.runtime) { into("libs/") }
     from(project(':streams:examples').jar) { into("libs/") }
     from(project(':streams:examples').configurations.runtime) { into("libs/") }
+    from(project(':streams:upgrade-system-tests-0100').jar) { into("libs/") }
```

Comment:
Could we double check if `build`, `install`, `releaseTarGz` and `uploadArchives` would not include this module, i.e. it will only be built upon running the system test?

Reply:
not sure why we should exclude from `build` or `install` ? Can you elaborate?

Reply:
Well, I guess it may not be too bad for `build` and `install`, just these are not necessary. Plus `install` will put the jar into the mvn cache which could be risky if we are relying on always build the jar from the current branch. Again, it is for cleanness more than correctness.